tests/functional_tests/test_protocols.py

```diff
@@ -14,46 +14,14 @@
 limitations under the License.
 """
 
-import os
-from pathlib import Path
-import pytest
 import logging
 
 import pandas as pd
 from numpy.testing import assert_allclose
 from pandas.testing import assert_frame_equal
 
-from bluepyemodel.evaluation.evaluation import get_evaluator_from_access_point
-from bluepyemodel.access_point import get_access_point
 
-TEST_ROOT = Path(__file__).parents[1]
-DATA = TEST_ROOT / "test_data"
-
-
-@pytest.fixture
-def api_config():
-    return {
-        "emodel": "cADpyr_L5TPC",
-        "emodel_dir": DATA,
-        "recipes_path": DATA / "config/recipes.json",
-    }
-
-
-@pytest.fixture
-def db(api_config):
-    return get_access_point("local", **api_config)
-
-
-@pytest.fixture
-def evaluator(db):
-
-    os.popen(f"nrnivmodl {DATA}/mechanisms").read()
-
-    db.get_mechanisms_directory = lambda: None
-    return get_evaluator_from_access_point(access_point=db)
-
-
-def test_protocols(db, evaluator):
+def test_protocols(db, evaluator, tmp_path):
```

Comment:
where is this tmp_path coming from?

Reply:
This is a fixture from pytest: https://docs.pytest.org/en/latest/how-to/tmp_path.html